### PR TITLE
Japanese Translation Tweak

### PR DESCRIPTION
**Pull request by EtoileRei**
- fixed typo
- improved readability
- standardized expression

### CODE REVIEW

**Comment by PabstMirror** [comment]
@classicarma - Do these changes look ok?

**Comment by classicarma** [comment]
@PabstMirror Almost perfect but I have added a comment to the translations.

File: addons/advanced_throwing/stringtable.xml [hunk 1]
```diff
@@ -160,7 +160,7 @@
         <Key ID="STR_ACE_Advanced_Throwing_dropModeHold">
             <English>Throwable Drop Mode (Hold)</English>
             <Russian>Режим броска гранаты (удерживать)</Russian>
-            <Japanese>投てきモード (押しっぱ)</Japanese>
+            <Japanese>投てきモード (押している間)</Japanese>
```
[the remark this answers is not in the document]
Comment (classicarma) [reply addons/advanced_throwing/stringtable.xml]: We talked about this and it's okay. please approve this pull-request

[the remark this answers is not in the document]
Comment (classicarma) [reply addons/advanced_throwing/stringtable.xml]: We talked about this and it's okay. please approve this pull-request

**Pull request merged**